2015-07-18 20:01:06,715 - [INFO] - from play in pool-4-thread-4 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2015-07-18 20:01:21,292 - [ERROR] - from application in New I/O worker #1 


! @6mobmjcp5 - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[inferred type arguments [Boolean] do not conform to method filter's type parameter bounds [T <: slick.lifted.Rep[_]]]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:01:21,445 - [WARN] - from play in New I/O worker #1 
No application found at invoker init

2015-07-18 20:02:47,013 - [ERROR] - from application in New I/O worker #1 


! @6mobmo9kk - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[inferred type arguments [Boolean] do not conform to method filter's type parameter bounds [T <: slick.lifted.Rep[_]]]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:04:48,042 - [ERROR] - from application in New I/O worker #1 


! @6mobn56lj - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[inferred type arguments [Boolean] do not conform to method filter's type parameter bounds [T <: slick.lifted.Rep[_]]]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:05:02,445 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:~/test

2015-07-18 20:05:02,682 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-07-18 20:06:40,294 - [ERROR] - from application in New I/O worker #1 


! @6mobnbgn1 - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[Cannot find an implicit ExecutionContext. You might pass
an (implicit ec: ExecutionContext) parameter to your method
or import scala.concurrent.ExecutionContext.Implicits.global.]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:07:01,352 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:~/test

2015-07-18 20:07:01,365 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-07-18 20:11:41,733 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:~/test

2015-07-18 20:11:41,754 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-07-18 20:11:55,451 - [ERROR] - from application in New I/O worker #1 


! @6mobo3f2g - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[Cannot write an instance of () => Unit to HTTP response. Try to define a Writeable[() => Unit]]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:12:22,484 - [ERROR] - from application in New I/O worker #1 


! @6mobo5329 - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[Cannot write an instance of Unit to HTTP response. Try to define a Writeable[Unit]]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:23:05,954 - [ERROR] - from application in New I/O worker #2 


! @6mobpfioo - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[constructor cannot be instantiated to expected type;
 found   : (T1, T2, T3, T4, T5)
 required: String]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

2015-07-18 20:23:52,967 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:~/test

2015-07-18 20:23:52,979 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-07-18 20:24:38,868 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:~/test

2015-07-18 20:24:38,875 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2015-07-18 20:24:53,510 - [ERROR] - from application in New I/O worker #2 


! @6mobplm20 - Internal server error, for (GET) [/play-auth/sub/login] ->

play.PlayExceptions$CompilationException: Compilation error[Cannot write an instance of Class[?0] to HTTP response. Try to define a Writeable[Class[?0]]]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at play.PlayExceptions$CompilationException$.apply(PlayExceptions.scala:27) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:49) ~[na:na]
	at play.PlayReload$$anonfun$taskFailureHandler$1.apply(PlayReload.scala:44) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.taskFailureHandler(PlayReload.scala:44) ~[na:na]
	at play.PlayReload$.compileFailure(PlayReload.scala:40) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at play.PlayReload$$anonfun$compile$1.apply(PlayReload.scala:17) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library-2.11.6.jar:na]
	at play.PlayReload$.compile(PlayReload.scala:17) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$2$$anonfun$apply$3$$anonfun$2.apply(PlayRun.scala:61) ~[na:na]
	at play.runsupport.Reloader.reload(Reloader.scala:295) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:122) ~[play_2.11-2.3.8.jar:2.3.8]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:120) ~[play_2.11-2.3.8.jar:2.3.8]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library-2.11.6.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library-2.11.6.jar:na]

